Exercise_C/index.css

```diff
@@ -1,4 +1,24 @@
-/* 
-This css sheet is linked to index.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+
+body{
+    margin:1px;
+    padding-left: 1px;
+   display: flex;
+   justify-content: space-between;
+}
+.square{
+    width: 60px;
+    height: 60px;
+    margin: 2px;
+}
+
+.a{
+    background-color: blue;
+}
+.b{
+    background-color: red;
+    position: relative;
+    top: 80px;
```

Comment:
Decent use of Flex.
To achieve this without using top 
you can define two flex containers and move a and c into the top and b into the bottom container, then be sure to apply flex
As before, consider using id.